ui/tests/acceptance/token-test.js

```diff
@@ -169,6 +169,9 @@ module('Acceptance | tokens', function(hooks) {
     const { oneTimeSecret, secretId } = managementToken;
 
     await JobDetail.visit({ id: job.id, ott: oneTimeSecret });
+
+    assert.notOk(currentURL().includes(oneTimeSecret), 'OTT is cleared from the URL after loading');
```

Comment:
Does this working imply that all our job detail tests are now 500ms longer?

Reply:
whew, good question… maybe it would make ALL acceptance tests longer since it’s in the application route! I’ll look more into this, thanks.

Reply:
The test audit action suggests it didn't. 🤷

Reply:
I don’t know… before I changed the default value from `null` to `''`, this was the first audit:

<img width="612" alt="image" src="https://user-images.githubusercontent.com/43280/114402358-5b443480-9b69-11eb-8815-6591167f880d.png">

The utility of it as it exists is in doubt to me, although I guess it would at least catch if something was DRAMATICALLY slower.

To hopefully reduce the variability from what happens on in CircleCI, I ran locally with 5 repetitions and it was _two minutes_ slower with the global delay vs the scoped delay I added in 7153b67. (`9m 23s 746ms` vs `7m 23s 150ms`) The unsightliness is increased within that commit, as it involves making the existence of the OTT query parameter checkable in `setupController` but it seems worth it to me to not drag on other tests. Thoughts? 😯

Reply:
Definitely worth it!